public/components/map_container/map_container.tsx

```diff
@@ -241,8 +241,10 @@ export const MapContainer = ({
       (indexPattern) => indexPattern.id === selectedLayerConfig.source.indexPatternId
     );
     if (!findIndexPattern) {
-      // @ts-ignore
-      const newIndexPattern = await indexPatterns.get(selectedLayerConfig.source.indexPatternId);
+      const newIndexPattern = await services.data.indexPatterns.get(
```

Comment:
I just couldn't match commit message and this code change.
`Correct indexPatterns api from services.data.indexPatterns to clean browser console error.`
but the code is changing indexPatterns to services.data.indexPatterns

Reply:
The method is used to update layersIndexPatterns which is only used in top nav comment as a requirement prop, but no use there currently(it's mainly used for global filter, we don't support yet).

It used to be passed in by:
```
    const {
      data: { indexPatterns },
    } = services;
```
so it's the reason we change from `indexPatterns ` to `services.data.indexPatterns`

https://github.com/opensearch-project/dashboards-maps/blob/2.8/public/components/map_top_nav/top_nav_menu.tsx#L141

Reply:
Got it. So it is `Use indexPatterns api from services.data.indexPatterns to clean browser console error.`